src/components/message/MessagesBox.tsx

```diff
@@ -124,7 +124,7 @@ const MessagesBox: React.FC<MessageProps> = (props: MessageProps) => {
         display: "flex",
         flexDirection: "column-reverse",
         width: "100%",
-        overflow: "scroll",
```

Comment:
`auto` because it will ensure no scroll bar is put on unless the view is getting bigger than its parent container. `overflowY` because we don't want horizontal scrolling here so if something goes wrong we should see things popping out and we can fix it.